fs/log.go

```diff
@@ -44,6 +44,25 @@ var logLevelToString = []string{
 	LogLevelDebug:     "DEBUG",
 }
 
+const (
+	// SystemdEmergPrefix is the string to prefix in Emergency for systemd-journald
```

Comment:
Does your systemd module have constants for these BTW?

Reply:
It does not. But it should actually, thanks for the idea ! I will update the lib and bump the version asap.